tests/unit/v0x01/test_symmetric/test_vendor_header.py

```diff
@@ -1,14 +1,14 @@
 """Testing VendorHeader message."""
 from pyof.v0x01.symmetric.vendor_header import VendorHeader
-from tests.unit.test_struct import TestStruct
+from tests.unit.test_struct import StructTest
+from pyof.v0x01.symmetric.hello import Hello
 
-
-class TestVendorHeader(TestStruct):
+class TestVendorHeader(StructTest):
     """Vendor message tests (also those in :class:`.TestDump`)."""
 
-    @classmethod
-    def setUpClass(cls):
-        super().set_minimum_size(8)
+    def setup_method(self):
+        self.set_raw_dump_object(Hello, xid=1)
```

Comment:
Added this line here from previous file [tests/unit/v0x01/test_symmetric/test_hello.py](https://github.com/kytos-ng/python-openflow/pull/90/files#diff-a3369021f2c46c4daea3ddd4a9515f2064fa7019969e958a1e06eb0d8672fa96). I noticed that with `unittest` it was also getting values from that file.